python/lsst/ip/diffim/imageDecorrelation.py

```diff
@@ -526,51 +525,51 @@ def setDefaults(self):
+      - @ref ip_diffim_imageDecorrelation_DecorrelateALKernelSpatialTask_Purpose
+      - @ref ip_diffim_imageDecorrelation_DecorrelateALKernelSpatialTask_Config
+      - @ref ip_diffim_imageDecorrelation_DecorrelateALKernelSpatialTask_Run
+      - @ref ip_diffim_imageDecorrelation_DecorrelateALKernelSpatialTask_Debug
+      - @ref ip_diffim_imageDecorrelation_DecorrelateALKernelSpatialTask_Example
 
-    \section ip_diffim_imageDecorrelation_DecorrelateALKernelSpatialTask_Purpose	Description
+    @section ip_diffim_imageDecorrelation_DecorrelateALKernelSpatialTask_Purpose	Description
 
     Pipe-task that removes the neighboring-pixel covariance in an
     image difference that are added when the template image is
     convolved with the Alard-Lupton PSF matching kernel.
 
-    This task is a simple wrapper around \ref DecorrelateALKernelTask,
+    This task is a simple wrapper around @ref DecorrelateALKernelTask,
     which takes a `spatiallyVarying` parameter in its `run` method. If
-    it is `False`, then it simply calls the `run` method of \ref
-    DecorrelateALKernelTask. If it is True, then it uses the \ref
+    it is `False`, then it simply calls the `run` method of @ref
+    DecorrelateALKernelTask. If it is True, then it uses the @ref
     ImageMapReduceTask framework to break the exposures into
-    subExposures on a grid, and performs the `run` method of \ref
+    subExposures on a grid, and performs the `run` method of @ref
     DecorrelateALKernelTask on each subExposure. This enables it to
     account for spatially-varying PSFs and noise in the exposures when
     performing the decorrelation.
 
-    \section ip_diffim_imageDecorrelation_DecorrelateALKernelSpatialTask_Initialize       Task initialization
+    @section ip_diffim_imageDecorrelation_DecorrelateALKernelSpatialTask_Initialize       Task initialization
 
-    \copydoc \_\_init\_\_
+    @copydoc \_\_init\_\_
 
-    \section ip_diffim_imageDecorrelation_DecorrelateALKernelSpatialTask_Run       Invoking the Task
+    @section ip_diffim_imageDecorrelation_DecorrelateALKernelSpatialTask_Run       Invoking the Task
 
-    \copydoc run
+    @copydoc run
 
-    \section ip_diffim_imageDecorrelation_DecorrelateALKernelSpatialTask_Config       Configuration parameters
+    @section ip_diffim_imageDecorrelation_DecorrelateALKernelSpatialTask_Config       Configuration parameters
 
-    See \ref DecorrelateALKernelSpatialConfig
+    See @ref DecorrelateALKernelSpatialConfig
 
-    \section ip_diffim_imageDecorrelation_DecorrelateALKernelSpatialTask_Debug		Debug variables
+    @section ip_diffim_imageDecorrelation_DecorrelateALKernelSpatialTask_Debug		Debug variables
 
     This task has no debug variables
 
-    \section ip_diffim_imageDecorrelation_DecorrelateALKernelSpatialTask_Example	Example of using DecorrelateALKernelSpatialTask
+    @section ip_diffim_imageDecorrelation_DecorrelateALKernelSpatialTask_Example  Example of using DecorrelateALKernelSpatialTask  # noqa E501
```

Comment:
Should there be two spaces before Example?

Reply:
I don't think it makes any difference. Unfortunately the `# noqa 501` is going to show up as part of the title so I have to fix that (in two places). Doxygen and pep8 don't get along very well. While I'm fixing that I'll make it one space in the files I fix.